### ISSUE TITLE
Add support for BASSOVAC

### ISSUE THREAD

**Issue by hammer** [hammerlab/biokepi#8]
https://github.com/ding-lab/bassovac

**Comment by hammer** [comment]
cc @iskandr

**Comment by smondet** [comment]
They don't seem to talk VCF. 
We need a converter, or yet another `<format> Biokepi_pipeline.t` for them, or we'll have to exclude them from `Biokepi_pipeline`.

**Comment by hammer** [comment]
Fair enough just wanted to file the issue as @iskandr had brought it up a while back. Sounds like a pain so I think we should push back and ensure there's value in getting it running.
